### ISSUE TITLE
Utilização somente do BoletoNetCore.PDF

### ISSUE THREAD

**Issue by luizcarlosucg** [BoletoNet/BoletoNetCore#54]
Pessoal, tudo bem?

Existe um meio de utilizar somente a emissão do PDF sem necessariamente preencher e validar todos os dados como se eu fosse gerar um arquivo de remessa?

Meu cenário é o seguinte: Eu tenho boletos já registrados no banco em carteiras/variações ainda não homologadas aqui no projeto. Tenho todos os dados já oficiais do boleto pelo banco, como código de barra e linha digitável.
Gostaria somente de gerar um PDF desse boleto.

Temos essa possibilidade utilizando somente o BoletoNetCore.PDF?

**Comment by olavorn** [comment]
Não, para criar uma boleto, sempre há a necessidade de executar o método de validação.
